### DISCUSSION TITLE
Free version to use of Fancybox

### DISCUSSION THREAD

**Discussion by Minuon** [fancyapps/ui#114] [Q&A]
I was just wondering if there are any free version to use of fancybox so that I can modify the script and use it on a non-profit/ profit site.

**Comment by fancyapps** [comment]
Hi,

Thank you for your interest. There are currently no plans to offer a free version. If you are a non-profit organization, I can offer you a coupon for a 50% discount. Contact me directly, you can find contact information on the website.

Janis
